src/index.tsx

```diff
@@ -1,23 +1,23 @@
-import React from 'react';
-import ReactDOM from 'react-dom/client';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import React from "react";
+import ReactDOM from "react-dom/client";
+import { BrowserRouter } from "react-router-dom";
+import "./index.css";
+import App from "./App";
+import reportWebVitals from "./reportWebVitals";
 
 // firebase
 import { initializeApp } from "firebase/app";
 import { getFirestore, connectFirestoreEmulator } from "firebase/firestore";
 import { getAuth, connectAuthEmulator } from "firebase/auth";
 import { getStorage, connectStorageEmulator } from "firebase/storage";
 
-
 const firebaseConfig = {
   apiKey: "AIzaSyCFj3LT5eKykepGw2udtDmYsmb5sBXlH9Y",
   authDomain: "astronomy-quiz-react.firebaseapp.com",
   projectId: "astronomy-quiz-react",
   storageBucket: "astronomy-quiz-react.appspot.com",
   messagingSenderId: "971999846040",
-  appId: "1:971999846040:web:31768aa241959e66453aed"
+  appId: "1:971999846040:web:31768aa241959e66453aed",
```

Comment:
I think we should not upload these confidential keys publically or on Github. You can add it to a different file and export it.

Reply:
thanks shraddha, did the changes